tenseal/tensealcontext.h

```diff
@@ -30,6 +29,7 @@ class TenSEALContext {
     shared_ptr<Encryptor> encryptor;
     shared_ptr<Decryptor> decryptor;
     shared_ptr<Evaluator> evaluator;
+    shared_ptr<TenSEALEncoder> encoder;
```

Comment:
Would it make sense to have a name like encoder_factory, encoder_store or simply encoders? I think it would help differentiate with encoder objects.

Reply:
Same thoughts here, I would like to suggest `encoder_factory`.